app/models/epic.rb

```diff
@@ -1,5 +1,5 @@
 class Epic < ApplicationRecord
   validates :name, presence: true
 
-  belongs_to :epic_list
+  has_and_belongs_to_many :epic_lists
```

Comment:
I would use has_many through instead of has_and_belongs_to_many (because we will add some attributes to join table in the future

Here is some nice article why not to you `has_and_belongs_to_many` association at all ;) 
https://flatironschool.com/blog/why-you-dont-need-has-and-belongs-to-many/